Pickers/AbstractActionSheetPicker.m

```diff
@@ -101,24 +100,24 @@ - (instancetype)init
     {
         self.presentFromRect = CGRectZero;
         self.popoverBackgroundViewClass = nil;
-
+        
         self.supportedInterfaceOrientations = (UIInterfaceOrientationMask) [[UIApplication sharedApplication]
-                                                                                           supportedInterfaceOrientationsForWindow:
-                                                                                                   [UIApplication sharedApplication].keyWindow];
-
+                                                                            supportedInterfaceOrientationsForWindow:
+                                                                            [UIApplication sharedApplication].keyWindow];
+        
```

Comment:
@BruceZCQ you still have unused trailing whitespaces in empty lines (in whole file). You have to remove them all.